WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -218,18 +268,37 @@ private static MockRpcClient CreateAndConfigureRpcClient(Func<int, FeeRate> esti
 					}),
 				OnGetPeersInfoAsync = async () =>
 					await Task.FromResult(hasPeersInfo ? new[] { new PeerInfo() } : Array.Empty<PeerInfo>()),
-
-				OnGetMempoolInfoAsync = async () =>
-					await Task.FromResult(new MemPoolInfo
-					{
-						MemPoolMinFee = memPoolMinFee // 1 s/b (default value)
-					}),
+				OnGetMempoolInfoAsync = async () => await Task.FromResult(ParseMemPoolInfo(memPoolMinFee)),
 				OnEstimateSmartFeeAsync = async (target, _) =>
 					await Task.FromResult(new EstimateSmartFeeResponse
 					{
 						Blocks = target,
 						FeeRate = estimator(target)
 					})
 			};
-	}
+
+		private static MemPoolInfo ParseMemPoolInfo(double memPoolMinFee)
+		{
+			var mempoolInfoWithHistogram = File.ReadAllText("./UnitTests/Data/MempoolInfoWithHistogram.json");
+			var jo = JObject.Parse(mempoolInfoWithHistogram);
+			var feeHistogram = (JObject)jo["fee_histogram"];
+
+			return new MemPoolInfo()
```

Comment:
I think it should be possible to do all this deserialization using JSON deserializer to avoid parsing it by hand. Unless there is a good reason to do so.

Reply:
That was my idea too until I started coding the the 50 FeeRateGroup instances with their 6 properties each and the header with all our code conventions what would require more than 400LoC so, I gave up.

Reply:
In the exhaustive test these instances are created programmatically with random number following tweaked normal  distribution but it is a good idea to have some more real example.